chia/consensus/block_creation.py

```diff
@@ -191,8 +191,9 @@ def create_foliage(
             tx_additions.append(coin)
             byte_array_tx.append(bytearray(coin.puzzle_hash))
         for coin in removals:
-            tx_removals.append(coin.name())
-            byte_array_tx.append(bytearray(coin.name()))
+            cname = coin.name()
```

Comment:
this is to save one call to `name()`

Reply:
I wouldn't have asked a comment to be added but if it is worth an explanation here in the context of an optimization PR then maybe it is worth a comment in the code.

Also, `coin_name` isn't that hard to write in three places.  `:]`

Reply:
the main reason I pointed it out here is because this change is (somewhat) unrelated to the main change. but it's a bit along the same theme